main.js

```diff
@@ -78,6 +78,13 @@ async function execute() {
         }
 
         const responseJson = {
+            // Version number of the file
+            version: 1,
```

Comment:
can you tell me more about your thought process here? what are you intending this to be used for? should we be updating it every time something new is released? is there a way to automate that? Bernard had some ideas I think, and Tessa too

Reply:
Every file should have a version number.  However, this is probably never going to change.

I would only increment this is there was a structural change to the way data is stored in the file.  For example, when you start running the scrapers independently, then the data structure might have to change to accomodate that.  Or, you might just decide that it would be better if the file were organized differently.

Then you can just put "if (verison === 2)" at the top of the client and assume that everything will work.  Otherwise, you are checking everywhere to see if the data that you need is available (like timestamps did).